.github/workflows/ci.yml

```diff
@@ -41,6 +41,9 @@ jobs:
       # Perform lint check
       - name: Run linter
         run: rush lint
+      # Perform formatting check
+      - name: Run prettifier
+        run: rush prettifier:check
```

Comment:
The reason we didn't have this extra step in before was that the eslint config has a prettier extension and covered this, just want to check this step is necessary

Reply:
(also in htmlstatic example package.json)

Reply:
I saw some minor differences, as in I had to run `prettifier --write` across the projects to make the check happy.

We might want to actually disable the eslint prettier extension. `eslint` is setup per project, while we want prettifier to be applied uniformly across projects (at least that's what our documentation claims)

Reply:
The only issue with disabling prettier in eslint is sometimes their rules conflict, however we'll probably have to pay the prettier cost twice if we have it as both its own check and in eslint

Reply:
(both CI runtime cost and cost that developers have to run and fix both prettier and eslint errors)

Reply:
Let me pull out this CI change for starters. We can always add that back in if it makes sense.

We already have this conflict in principle: the pre-commit hook forces you to fix prettier errors, while CI uses the eslint-prettier plugin. And I found that the didn't quite agree. My guess is that it's because the per-project eslint rules didn't have the same prettier config as the top-level one.